Makefile

```diff
@@ -281,6 +281,7 @@ kind-up: $(KIND)
 kind-down: $(KIND)
 	$(KIND) delete cluster --name gardener-local
 	rm -f $(REPO_ROOT)/example/provider-local/base/kubeconfig
+	rm -rf dev/etcd-local
```

Comment:
Just a note here, the actual backupbucket will be in a folder like this: `97511b4d-0de3-4d5e-b5f1-5a1810641538` so we can support multiple backupbuckets and also test copying of etcd backups. So I guess it would be better to call the folder that we create for the backups like this:
```
dev/local-backupbuckets
```
or
```
dev/local-backup-storage
```